src/BlockTypes/AbstractBlock.php

```diff
@@ -143,4 +143,37 @@ protected function enqueue_data( array $attributes = [] ) {
 	protected function enqueue_scripts( array $attributes = [] ) {
 		// noop. Child classes should override this if needed.
 	}
+
+	/**
+	 * Script to append the correct sizing class to a block skeleton.
+	 *
+	 * @return string
+	 */
+	protected function get_skeleton_inline_script() {
+		return "<script>
+			const containers = document.querySelectorAll( 'div.wc-block-skeleton' );
```

Comment:
Nit: `containers` is a const while `w` and `classname`, defined some lines below, are vars. :point_down: 

I think it would be nice to keep it consistent and either always use var or always const/let. I don't have any preference, it looks like [const](https://caniuse.com/#feat=const) and [let](https://caniuse.com/#feat=let) support in all browsers including IE11 is good enough.